docs/package/gluon-config-mode-geo-location.rst

```diff
@@ -0,0 +1,55 @@
+gluon-config-mode-geo-location
+==============================
+
+This package allows the user to set latitude, longitude and optionally altitude
+within config mode. There are 2 types of this package.
+
+It is possible to include ``gluon-config-mode-geo-location`` or
+``gluon-config-mode-geo-location-with-map`` in the ``site.mk``.
+
+gluon-config-mode-geo-location-with-map
+---------------------------------------
+
+The package ``gluon-config-mode-geo-location-with-map`` will additionally and a
+map where users can pick a position. This map will only shown if the users
+computer will have an internet connection.
+
+site.conf
+^^^^^^^^^
+
+This option is valid for both typ of package.
```

Comment:
"typ of package" -> "package variants"  <-- not sure though about the meaning

Reply:
Your version sounds better.